fetchcode/purl2url/__init__.py

```diff
@@ -0,0 +1,134 @@
+# fetchcode is a free software tool from nexB Inc. and others.
+# Visit https://github.com/nexB/fetchcode for support and download.
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and http://aboutcode.org
+#
+# This software is licensed under the Apache License version 2.0.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at:
+# http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+from urllib.parse import urlparse
+
+
+def index_in_list(index, list):
+    return index < len(list)
+
+
+def build_bitbucket(name, namespace, version, qualifiers, subpath):
```

Comment:
Can we add some descriptive docstring for this function

Reply:
Got it